src/main/clojure/jarl/builtins/conversions.clj

```diff
@@ -20,43 +17,37 @@
 ; Deprecated
 
 (defn builtin-cast-array
-  "Implementation of cast_array built-in"
-  {:builtin "cast_array" :args-types [(sorted-set "array" "set")]}
   [{[x] :args}]
-  (check-args (meta #'builtin-cast-array) x)
-  (vec x))
+  (if (or (vector? x) (set? x))
+    (vec x)
+    (throw (errors/type-ex "cast_array: operand 1 must be one of {array, set}"))))
 
 (defn builtin-cast-set
-  "Implementation of cast_set built-in"
-  {:builtin "cast_set" :args-types [(sorted-set "array" "set")]}
   [{[x] :args}]
-  (check-args (meta #'builtin-cast-set) x)
-  (set x))
+  (if (or (vector? x) (set? x))
+    (set x)
+    (throw (errors/type-ex "cast_set: operand 1 must be one of {array, set}"))))
 
 (defn builtin-cast-string
-  "Implementation of cast_string built-in"
-  {:builtin "cast_string" :args-types ["string"]}
   [{[x] :args}]
-  (check-args (meta #'builtin-cast-string) x)
-  x)
+  (if (string? x)
+    x
+    (throw (errors/type-ex "cast_boolean: operand 1 must be boolean"))))
```

Comment:
`cast_string`, I'm guessing.

Reply:
Also guessing this isn't covered by a test :p.

Reply:
haha! no, I don't think these are meant to be used :) I'll fix the names though.